drivers/virtualbox/virtualbox.go

```diff
@@ -381,12 +379,10 @@ func (d *Driver) Start() error {
 		if err := vbm("startvm", d.MachineName, "--type", "headless"); err != nil {
 			return err
 		}
-		log.Infof("Starting VM...")
```

Comment:
IMO these types of messages (ones which might overlap / duplicate what the core is saying) should stay out of the main output, maybe we could put them in `Debug` if we _really_ want to keep them.